www/docs/style/sass/pages/_mp.scss

```diff
@@ -790,4 +789,26 @@ a[href^="https://www.publicwhip.org"] {
 .postcode-mp-image-wrapper {
     display: block;
     margin-top: 5px;
+}
+
+.interest-summary {
+    font-weight: bold;
+    margin-top: 1em;
+    margin-bottom: 1em;
+}
+
+.interest-child-items > div.interest-item > p.interest-summary {
+    font-weight: normal;
+    text-decoration: none;
+}
+
+.interest-details-list {
+    font-size: 0.8em;
+}
+
+.child-item-header {
+    text-decoration: underline;
```

Comment:
This doesn't feel to visually fit - if you want a smaller heading then I'd make it bold and smaller font (or make the other one bigger), we need a heading structure. Or indent it maybe. (Also, with conversation in other PR, really this should be a hN if it's a heading, not a div, but I realise that might not be doable now.)
Feels like all the metadata could be hidden behind a details element and only shown if requested? It reads quite dry/technical (ISO dates, repetition of amount without pound symbol, etc). And then it wouldn't need the heading so much.

Reply:
Based on revised heading structure from other PR: ![image](https://github.com/user-attachments/assets/f84e992a-c832-412c-bedd-d55b325f0394)

Agree on the detail bit but I think that needs to wait for another pass - ideally we want to be passing storing the json in the database and get the formatting into our normal template (it's in three places at the moment)

Reply:
Yep!